### DISCUSSION TITLE
父子应用都有Onglobalstatechange，父应用setglobalstate之后，触发了父应用的change事件，没有触发子应用的

### DISCUSSION THREAD

**Discussion by lixioo** [umijs/qiankun#2489] [Q&A]
父子应用都有Onglobalstatechange，父应用setglobalstate之后，触发了父应用的change事件，没有触发子应用的，你们有遇到过吗，子的change事件只在初始化的触发过一次，后续父应用进行set，子的就没有触发了

**Comment by JackDan9** [comment]
贴段代码
